From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 21 May 2020 12:27:29 -0400
Subject: [PATCH] disable media DRM preprovisioning by default

This switches to fetching on-demand, which can only happen if DRM media
support is enabled.
---
 media/base/media_switches.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
index b8f748b183652..b6bf9d2583e28 100644
--- a/media/base/media_switches.cc
+++ b/media/base/media_switches.cc
@@ -1002,7 +1002,7 @@ BASE_FEATURE(kMediaDrmPersistentLicense,
 // which will trigger provisioning process after MediaDrmBridge is created.
 BASE_FEATURE(kMediaDrmPreprovisioning,
              "MediaDrmPreprovisioning",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Determines if MediaDrmOriginIdManager should attempt to pre-provision origin
 // IDs at startup (whenever a profile is loaded). Also used by tests that
